pkg/istio/rbac/v1/provider.go

```diff
@@ -96,15 +101,167 @@ func (p *v1) ConvertAthenzModelIntoIstioRbac(m athenz.Model) []model.Config {
 // GetCurrentIstioRbac returns the ServiceRole and ServiceRoleBinding resources for the specified model's namespace
 func (p *v1) GetCurrentIstioRbac(m athenz.Model, csc model.ConfigStoreCache) []model.Config {
 
-	sr, err := csc.List(model.ServiceRole.Type, m.Namespace)
+	sr, err := csc.List(collections.IstioRbacV1Alpha1Serviceroles.Resource().GroupVersionKind(), m.Namespace)
 	if err != nil {
 		log.Errorf("Error listing the ServiceRole resources in the namespace: %s", m.Namespace)
 	}
 
-	srb, err := csc.List(model.ServiceRoleBinding.Type, m.Namespace)
+	srb, err := csc.List(collections.IstioRbacV1Alpha1Servicerolebindings.Resource().GroupVersionKind(), m.Namespace)
 	if err != nil {
 		log.Errorf("Error listing the ServiceRoleBinding resources in the namespace: %s", m.Namespace)
 	}
 
 	return append(sr, srb...)
 }
+
+// ConvertAthenzModelIntoIstioAuthzPolicy converts the Athenz RBAC model into Istio Authorization V1Beta1 specific
```

Comment:
Is it possible to modify the `ConvertAthenzModelIntoIstioRbac` function to work with `ConvertAthenzModelIntoIstioAuthzPolicy` so that we can keep the same interface function?